ui/frontend/index.scss

```diff
@@ -195,18 +195,25 @@ body {
         border-left: $border;
         background: none;
 
-        cursor: pointer;
+        &:enabled {
```

Comment:
I already had this, but why did I... buttons already set the cursor. Will have to look into that in the future.